testcharms/charm-repo/quantal/metered-multi-series/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+name: metered-multi-series
+summary: "A metered charm with custom metrics"
+description: ""
+series:
+  - xenial
```

Comment:
If we are saying it is multi-series, perhpas pub multiple series here? Add trusty.